charts/dev/theme-park/values.yaml

```
@@ -0,0 +1,36 @@
+image:
+  repository: ghcr.io/k8s-at-home/theme-park
+  pullPolicy: IfNotPresent
+  tag: v1.9.4@sha256:6d576b5aa9c08305026ff76ba9e08e47c40dfce958293f5b3307cae05cff873a
+
+service:
+  main:
+    ports:
+      main:
+        port: 8080
```

```suggestion
        port: 10244
```

We use incremental ports to avoid conflicts when users, just next next next.

Currently the list with the ports is slightly outdated.
https://truecharts.org/manual/default-ports/

But we'll soon have it autogenerated.